wiki/Rules/id.md

```diff
@@ -18,7 +18,7 @@ Peraturan-peraturan berikut ini merupakan pedoman dasar yang kami terapkan di li
-6. **Bersikaplah bijak seputar hal-hal yang hendak kamu bagikan kepada orang lain, terutama jika hal-hal tersebut menyangkut konten yang dapat mengganggu kenyamanan umum.** Apabila kamu harus berpikir lebih dari satu kali sebelum mengimbuhkan konten yang ingin kamu kirim, besar kemungkinan konten tersebut memang kurang layak untuk dikirim. Pada umumnya, sebisa mungkin hindari mengirim hal-hal yang dapat memicu epilepsi, mengandung unsur kekerasan, atau konten-konten sensitif lainnya ke dalam ruang publik.
+6. **Bersikaplah bijak seputar hal-hal yang hendak kamu bagikan kepada orang lain, terutama apabila hal-hal tersebut menyangkut konten-konten yang dapat mengganggu kenyamanan umum.** Apabila kamu harus berpikir lebih dari satu kali sebelum mengimbuhkan konten yang ingin kamu kirim, besar kemungkinan konten tersebut memang kurang layak untuk dikirim. Pada umumnya, sebisa mungkin hindari mengirim hal-hal yang dapat memicu epilepsi, mengandung unsur kekerasan, atau konten-konten sensitif lainnya ke dalam ruang publik.
 7. **Jaga tautan hanya pada situs-situs web yang berstatus legal.** Distribusi konten-konten video (di samping melalui situs-situs web yang telah berlisensi resmi seperti Animelab dan Crunchyroll) atau perangkat lunak/"warez" milik pihak lain secara ilegal dalam bentuk apapun tidak dibenarkan di dalam lingkungan osu!. Tautan siaran Twitch non-osu! HANYA dibenarkan untuk disebar pada kanal `#videogames`.
 8. **Jangan mengiklankan server chat milik pihak ketiga di luar osu! dalam bentuk apapun - baik itu server Discord, Teamspeak, Skype, ataupun server-server chat lainnya.** Obrolan osu! diutamakan untuk membahas berbagai topik pembicaraan yang terkait dengan osu!, bukan sebagai sarana promosi publik untuk membujuk orang-orang agar bergabung ke dalam server pribadimu. Kirimkanlah pesan pribadi secara khusus apabila kamu memang ingin mengajak seseorang untuk masuk ke dalam server milikmu.
 9. **Pada kanal-kanal yang dikhususkan secara spesifik untuk suatu komunitas kenegaraan, kamu harus turut berbicara dalam bahasa yang dominan digunakan dalam kanal tersebut (bahasa Inggris untuk kanal `#english`, bahasa Thailand untuk kanal `#thai`, dll).** Teruntuk kanal-kanal yang tidak terikat secara khusus kepada suatu bahasa, pada umumnya percakapan akan selalu dilangsungkan dalam bahasa Inggris. Meskipun demikian, terdapat pengecualian pada kanal `#lobby` dan `#help` di mana kamu dibenarkan untuk mengobrol dalam bahasa apapun yang kamu kehendaki pada kedua kanal ini. *Anggota [moderation staff](/wiki/People/The_Team/Global_Moderation_Team) memiliki hak khusus untuk dapat berbicara bahasa apapun pada kanal manapun apabila dirasa perlu selama masa jabatan mereka masih berlaku.*
```

Comment:
bagaimana `#thai` diganti sama `#indonesian` karena emang ini terjemahannya indonesia ?

lalu untuk `*Anggota [moderation staff]` mau diganti ke `*Anggota [staf administratif osu!]` ??

Reply:
improved the wordings on line 21 slightly

i'm not sure about the other suggestions tho, especially since not all GMTs are being listed as "general housekeepers" in http://osu.ppy.sh/help/wiki/People/The_Team...